src/components/author/author.twig

```diff
@@ -63,14 +63,16 @@
   <div class="c-author__content-container o-media__content">
     {# Author links #}
     <p>
-      <span class="u-hidden-visually">{{ author_prefix|default("By") }}</span>
+      <span class="u-hidden-visually" role="text">
+        {{ author_prefix|default("By") }}
+      </span>
       {% block authors %}
         {% for author in authors %}
           {% if loop.last and loop.length > 1 %}and {% endif %}
           {%- if author.link and not unlink -%}
             <a class="c-author__author" href="{{ author.link }}">{{ author.name }}</a>
           {%- else -%}
-            <span class="c-author__author">{{ author.name }}</span>
+            <span class="c-author__author" role="text">{{ author.name }}</span>
```

Comment:
Hmm, if we add a space here it reads "by" and the first name as two separate words:

```suggestion
            <span class="c-author__author" role="text"> {{ author.name }}</span>
```

But, it also messes with it visually, so I'm not sure this is what we want